community/blog/2021-12 2021, the year that Arcus matured/README.md

```diff
@@ -0,0 +1,173 @@
+# 2021, the Year Arcus Matured
+And here we are again. A year has passed and a lot of changes have been made on the Arcus front. This post will summarize the biggest changes that have been made in the year 2021.
+
+2021 was a very big year for Arcus. In every repository major changes were made. This resulted in full production-ready open-source libraries that are ready to be used within your projects.
```

Comment:
```suggestion
2021 was an impactfull year for Arcus. In every repository major changes were made. This resulted in full production-ready open-source libraries that are ready to be used within your projects.
```

Reply:
Should probably be `impactful` instead of `impactfull` 🤓